fdbt-dev/data/cappedMatchingData/cappedMultiServiceByTaps.json

```diff
@@ -0,0 +1,47 @@
+{
+  "nocCode": "BLAC",
+  "type": "capped",
+  "passengerType": { "id": 1 },
+  "email": "test@example.com",
+  "uuid": "BLAC75196528",
+  "ticketPeriod": { "startDate": "2020-02-01T00:00:00.000Z" },
+  "selectedServices": [
+    {
+      "lineName": "2",
+      "lineId": "YpQjUw",
+      "serviceCode": "NW_05_BLAC_2_1",
+      "serviceDescription": "POULTON - BLACKPOOL via Victoria Hospital Outpatients"
+    },
+    {
+      "lineName": "2C",
+      "lineId": "vySmfewe0",
+      "serviceCode": "NW_05_BLAC_2C_1",
+      "serviceDescription": "KNOTT END - POULTON - BLACKPOOL"
+    }
+  ],
+  "cappedProductInfo": {
+    "tapDetails": { "0": "2", "1": "2", "2": "1" },
+    "capDetails": {
+      "caps": [
+        {
+          "name": "Day cap",
+          "price": "2",
+          "durationAmount": "2",
+          "durationUnits": "day"
+        },
+        {
+          "name": "wefeFw",
+          "price": "2",
+          "durationAmount": "1",
+          "durationUnits": "day"
+        }
+      ],
+      "productName": "nameyname"
```

Comment:
can we have Multi service product by taps

Reply:
name changed